docs/en/api/serial.rst

```diff
@@ -23,6 +23,10 @@ with additional features for advanced use cases.
 * **Event callbacks**: Receive and error event callbacks
 * **Configurable buffers**: Adjustable RX and TX buffer sizes
 
+.. note::
+   In case that both pins RX and TX are detached from UART, the driver will be stopped.
```

Comment:
In the case of both pins, RX and TX are detached from the UART, the driver will be stopped.